src/applications/vaos/components/FacilityAddress.jsx

```diff
@@ -17,6 +21,9 @@ export default function FacilityAddress({ name, address, phone }) {
       {!!address.address3 && <br />}
       {address.city}, {address.state} {address.zip}
       <br />
+      <FacilityDirectionsLink location={facility} />
```

Comment:
I think it's fine, but this link will also show up on the clinic choice page, since FacilityAddress is used there, too.